contrib/opencensus-ext-azure/tests/test_azure_standard_metrics.py

```diff
@@ -46,3 +47,38 @@ def test_get_available_memory(self, psutil_mock):
         mem = standard_metrics.get_available_memory()
 
         self.assertEqual(mem, 100)
+
+    def test_get_process_private_bytes_metric(self):
+        gauge = standard_metrics.get_process_private_bytes_metric()
+
+        self.assertEqual(gauge.descriptor.name,
```

Comment:
Add a TODO here. Need to evolve the backend to be more generic (currently it is "Windows focus" when one look at the names).